awscli/customizations/ec2decryptpassword.py

```diff
@@ -79,11 +82,11 @@ def add_to_params(self, parameters, value):
             path = os.path.expanduser(path)
             if os.path.isfile(path):
                 self._key_path = path
-                service_name = self._operation.service.endpoint_prefix
+                service_name = \
+                    self._operation_model.service_model.service_name
```

Comment:
Updated.  Don't think it matters in this specific scenario (`endpoint_prefix == service_name` for ec2), but we should be consistent.